manifests/h/HydrusNetwork/HydrusNetwork/505b/HydrusNetwork.HydrusNetwork.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: HydrusNetwork.HydrusNetwork
+PackageVersion: 505b

```

```suggestion
PackageVersion: v505b
```
This must match Control Panel.